batch10/assignment2/tamkeen-zarah.md

```diff
@@ -36,7 +36,7 @@
 
 # Q4) Write command to create an nginx container in detached mode with name assignment-2 running on host port 9090 and container port 80 on a custom network named assignment-2
 
-sudo docker network create 9090:80
+sudo docker network create network1
 
 sudo docker container run --net 9090:80 assignment-2 
```

Comment:
The network command is correct now but the command to run containers is wrong, the network name should be after --net and the port should be with -p flag